File: Arrays/TwoSum.java
package Arrays;

public class TwoSum {
    public int[] twoSum(int[] nums, int target) {
        int n = nums.length;
        for(int i = 0; i < n; i ++){
            for(int j = i + 1; j < n; j++){
                if (nums[i] + nums[j] == target){
                    return new int[]{i, j};
                }
            }
        }
        return new int[]{};
    }

    public static void main(String[] args) {
        TwoSum ts = new TwoSum();
        int[] nums = {2, 7, 11, 15};
        int target = 9;
        int[] result = ts.twoSum(nums, target);
        for (int i : result) {
            System.out.print(i + " ");
        }
    }
}

File: Practice/TwoSum.py
def twosum(arr, target):
    n = len(arr)
    for i in range(n):
        for j in range(i + 1, n):
            if arr[i] + arr[j] == target:
                return (i, j)
    
    
# Example usage
arr = [2, 7, 11, 15]
target = 9
result = twosum(arr, target)
print(f"Indices of the two numbers that add up to {target} are: {result}")
    